website/docs/backends/types/pg.html.md

```diff
@@ -71,6 +71,7 @@ The following configuration options or environment variables are supported:
 
  * `conn_str` - (Required) Postgres connection string; a `postgres://` URL
  * `schema_name` - Name of the automatically-managed Postgres schema, default `terraform_remote_state`.
+ * `skip_schema_creation` - If set to `true`, Terraform won't try to create the Postgres schema.
```

Comment:
Please note why this might be useful too, like:

```suggestion
 * `skip_schema_creation` - If set to `true`, the Postgres schema must already exist. Terraform won't try to create the schema. Useful when the Postgres user does not have "create schema" permission on the database.
```